### PR TITLE
ADH-426 port Slider 0.98 to bigtop (rpm only)

### PR DESCRIPTION
**Pull request by deart2k**

### CODE REVIEW

File: bigtop-packages/src/common/slider/do-component-build [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+
+. `dirname $0`/bigtop.bom
+
+mvn clean site:site site:stage package -DskipTests
```
Comment (acmnu) [inline bigtop-packages/src/common/slider/do-component-build]: No new line at the end.

File: bigtop-packages/src/common/slider/install_slider.sh [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+tar xf "slider-assembly/target/slider-${version}-incubating-all.tar.gz" -C "${prefix}/usr/lib/"
+
+mv "${prefix}/usr/lib/slider-${version}-incubating" "${slider_dir}"
+
+mv "${slider_dir}/conf" "${etc_dir}/conf"
+cp -r "${etc_dir}/conf" "${etc_dir}/conf.dist"
+
+
+
+ln -nsf "/etc/slider/conf" "${slider_dir}/conf"
```
Comment (acmnu) [inline bigtop-packages/src/common/slider/install_slider.sh]: No new line at the end. New IDE?

**Review by acmnu — CHANGES_REQUESTED** [review]

File: bigtop-packages/src/common/slider/do-component-build (hunk 1 above)
Comment (deart2k) [reply bigtop-packages/src/common/slider/do-component-build]: added new line at the end of file

File: bigtop-packages/src/common/slider/install_slider.sh (hunk 2 above)
Comment (deart2k) [reply bigtop-packages/src/common/slider/install_slider.sh]: mcedit.  added new line at the end of file

**Review by acmnu — APPROVED** [review]

**Pull request merged**